test/test_ex2_obc_software/ex2_system/source/logger/test_logger.c

```diff
@@ -27,6 +27,12 @@ AfterEach(logger) {};
 int32_t red_open( const char *pszPath, uint32_t ulOpenMode) {
     return mock(pszPath, ulOpenMode);
 }
+int32_t red_rename(
```

Comment:
The reason cGreen is failing is because you didn't make mocks for red_transact, red_read, and red_unlink like you did here for red_rename